packages/ts/react-crud/src/data-provider.ts

```diff
@@ -121,6 +125,74 @@ export abstract class DataProvider<TItem> {
   protected abstract fetchFilteredCount(page: DataPage<TItem>): Promise<number | undefined> | number | undefined;
 }
 
+export abstract class AbstractComboBoxDataProvider<TItem> {
+  protected readonly list: ComboBoxFetchCallback<TItem>;
+  protected readonly loadTotalCount?: boolean;
+
+  protected sort: Sort | undefined;
+  protected totalCount: number | undefined;
+  protected filteredCount: number | undefined;
+
+  constructor(list: ComboBoxFetchCallback<TItem>, sort: Sort | undefined) {
+    this.list = list;
+    this.load = this.load.bind(this);
```

Comment:
Since we have got rid of the unbound function, we probably don't need to do this binding